app/src/main/res/layout/dialog_choose_avatar_source.xml

```diff
@@ -20,8 +20,6 @@
       android:drawableStart="@drawable/ic_camera"
       android:gravity="center_vertical"
       android:text="@string/upload_dialog_photo"
-      android:textColor="@color/create_store_title_color"
-      android:textSize="@dimen/text_size_medium_small"
```

Comment:
was the removal of textColor intentional here? The style that it has is only defining size and not color. I understand it will use the default color but should we add the style with the correct text color ?

Reply:
solved